app/src/main/java/org/oppia/app/topic/TopicFragment.kt

```diff
@@ -6,18 +6,22 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import org.oppia.app.fragment.InjectableFragment
+import org.oppia.domain.topic.TEST_TOPIC_ID_0
 import javax.inject.Inject
 
 /** Fragment that contains tabs for Topic. */
 class TopicFragment : InjectableFragment() {
-  @Inject lateinit var topicFragmentPresenter: TopicFragmentPresenter
+  @Inject
+  lateinit var topicFragmentPresenter: TopicFragmentPresenter
+  lateinit var topicId: String
 
   override fun onAttach(context: Context?) {
     super.onAttach(context)
     fragmentComponent.inject(this)
   }
 
   override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-    return topicFragmentPresenter.handleCreateView(inflater, container)
+    topicId = arguments?.getString(TOPIC_ID_ARGUMENT_KEY) ?: TEST_TOPIC_ID_0
```

Comment:
**NOTE**: `TEST_TOPIC_ID_0` will be removed in either #285 or a separate PR. For now we need this successfully running the test-cases.